client/src/components/sidebar/Sidebar.tsx

```diff
@@ -179,84 +198,99 @@ const Sidebar: React.FC = () => {
+              </a>
+              {!collapsed && (
+                <>
+                  <SidebarTitle variant="h6">Notangles</SidebarTitle>
+                </>
+              )}
+            </HeaderContainer>
 
-          <Divider />
+            <Divider />
 
-          <SideBarContainer>
+            <SideBarContainer>
+              <NavComponentsContainer>
+                <CustomModal
+                  title="Timetable"
+                  toolTipTitle="Timetable"
+                  showIcon={<CalendarMonth />}
+                  description={'Current Timetable'}
+                  content={null}
+                  collapsed={collapsed}
+                  // currently not clickable since this is our current page
+                  isClickable={false}
+                  // hardcoded until we move away from single page site
+                  isSelected={true}
+                />
+                <FriendsButton collapsed={collapsed} />
+                <Divider />
+                {modalData.map((modal, index) => (
+                  <React.Fragment key={index}>
+                    <CustomModal
+                      title={modal.title}
+                      toolTipTitle={modal.toolTipTitle}
+                      showIcon={modal.showIcon}
+                      description={modal.description}
+                      content={modal.content}
+                      collapsed={collapsed}
+                      isClickable={modal.isClickable}
+                    />
+                  </React.Fragment>
+                ))}
+              </NavComponentsContainer>
+            </SideBarContainer>
+          </div>
 
-            <NavComponentsContainer>
-              <CustomModal
-                title="Timetable"
-                toolTipTitle="Timetable"
-                showIcon={<CalendarMonth />}
-                description={'Current Timetable'}
-                content={null}
-                collapsed={collapsed}
-                // currently not clickable since this is our current page
-                isClickable={false}
-                // hardcoded until we move away from single page site
-                isSelected={true}
-              />
-              <FriendsButton collapsed={collapsed} />
-              <Divider />
-              {modalData.map((modal, index) => (
-                <React.Fragment key={index}>
-                  <CustomModal
-                    title={modal.title}
-                    toolTipTitle={modal.toolTipTitle}
-                    showIcon={modal.showIcon}
-                    description={modal.description}
-                    content={modal.content}
-                    collapsed={collapsed}
-                    isClickable={modal.isClickable}
-                  />
-                </React.Fragment>
+          <SidebarFooter>
+            <DarkModeButton collapsed={collapsed} />
+            <UserAccount collapsed={collapsed} />
+            {!isMobile &&
+              (!collapsed ? (
```

Comment:
no need for extra bracket
```suggestion
              !collapsed ? (
```